components/finder.rst

```diff
@@ -329,6 +329,12 @@ it is called with the file as a :class:`Symfony\\Component\\Finder\\SplFileInfo`
 instance. The file is excluded from the result set if the Closure returns
 ``false``.
 
+.. versionadded:: 6.4
+
+Since Symfony 6.4 filter can early prune directories, pass ``true`` as a second
+``filter()`` method parameter and when the Closure returns ``false`` the deeper
```

Comment:
```suggestion
Since Symfony 6.4, a filter can prune directories early, pass ``true`` as a second
parameter for the ``filter()`` method and when the Closure returns ``false`` the deeper
```